portfolio/src/main/webapp/index.html

```diff
@@ -137,6 +137,24 @@ <h3 class="title is-3">Projects</h3>
             </div>
         </section>
 
+        
+        <section id="servlet" class="section section-5">
+            <div class="container">
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#Indentation

2 space indentation

Reply:
Thanks for pointing that out. I will revise the indentation.